pkg/specgenutil/volumes.go

```diff
@@ -525,11 +525,11 @@ func getBindMount(args []string) (spec.Mount, error) {
 	return newMount, nil
 }
 
-// Parse a single tmpfs mount entry from the --mount flag
-func getTmpfsMount(args []string) (spec.Mount, error) {
+// Parse a single tmpfs/ramfs mount entry from the --mount flag
+func getTmpfsMount(args []string, mountType string) (spec.Mount, error) {
```

Comment:
Can we rename the function to `parseMount` or something like that?  It's not strictly `Tmpfs` only anymore.

Reply:
Add noswap, but this seems like it is only supported in rootful mode.